website/views.py

```diff
@@ -8,8 +8,9 @@
 from flask_login import login_required
 from sqlalchemy import or_, and_
 from website import db
-from .models import Appointment, ApplicantInformation, ApplicantPreferences, AdditionalInformation, AssessmentForm
+from .models import Appointment, ApplicantInformation, ApplicantPreferences, AdditionalInformation, AssessmentForm, ApplicantResume
```

Comment:
This line throws an error, because it ApplicantResume has not been put into models yet

Reply:
The other pull request should solve the issue of the ApplicantResume once merged